src/sensor/ping360.h

```diff
@@ -421,6 +470,7 @@ class Ping360 : public PingSensor
 
     QElapsedTimer _messageElapsedTimer;
     QTimer _timeoutProfileMessage;
+    QTimer _preConfigurationTimer;
```

Comment:
```
/**
     * @brief This timer allows us to wait for a couple of seconds for an answer.
     *  If the baudrate is not valid or the sensor is unable to communicate with us because of noise or something else,
     *  we should trigger the request again, since it forces the request to be made by our side.
     *  Without this timer, we are going to wait forever for a reply without asking again.
     */
    QTimer _preConfigurationTimer;
```